djangocms_version_locking/cms_config.py

```diff
@@ -9,7 +9,7 @@
 class VersionLockingCMSExtension(CMSAppExtension):
```

Comment:
as Version Locking is dependent on Versioning, I feel we will need to do something similar that is done in moderation https://github.com/divio/djangocms-moderation/blob/release/1.0.x/djangocms_moderation/cms_config.py#L16